### ISSUE TITLE
Include a link to the hosted documentation in the repository details

### ISSUE THREAD

**Issue by christianscott** [bazelbuild/rules_pkg#881]
It's a little bit tricky to find the docs currently. I didn't realise there were any until I found them via google search. You can add a link to the repo details by clicking the gear (⚙️) icon next to "about" on the LHS of the repo homepage

**Comment by aiuto** [comment]
I'm not sure what you mean. The links to the docs is in the home page README.

**Comment by christianscott** [comment]
yep I can see that but it's easy to miss. this would be a quick way to make it easier for newbies to find the docs

example here in the bazel repo: https://github.com/bazelbuild/bazel

![Screenshot 2024-08-13 at 2 46 35 PM](https://github.com/user-attachments/assets/fd7c1dfd-8f36-4a61-85b6-a6c1e45d4b3f)

**Comment by aiuto** [comment]
Oh. That.  I never would have noticed it. I ignore most boiler-plate.  The about box is one I rarely look in.  I can see about getting one of the admins go do something.

It still sort of feels funny.  A link like that should be to the project rather than a specific part. The bazel.build link is to the overall project - not really the docs. You have to hunt the bazel project site to find the reference pages.
